### PR TITLE
add migration and orm changes for additional lgs fields and delegate

### PR DESCRIPTION
**Pull request by jinhoonbang**

### CODE REVIEW

**Review by samsondav — COMMENTED** [review]
What is legacy gas station?

**Comment by jinhoonbang** [comment]
> What is legacy gas station?

synced offline

File: go.mod [hunk 1]
```diff
@@ -382,4 +385,8 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
+
+	github.com/smartcontractkit/capital-markets-projects => /Users/jinbang/development/capital-markets-projects
+
+	github.com/smartcontractkit/capital-markets-projects/lib => /Users/jinbang/development/capital-markets-projects/lib
 )
```
Comment (jinhoonbang) [inline go.mod:392]: waiting for corresponding capital-markets-projects repo changes to be merged

File: core/config/docs/core.toml [hunk 2]
```diff
@@ -13,6 +13,8 @@ FeedsManager = true # Default
 LogPoller = false # Default
 # UICSAKeys enables CSA Keys in the UI.
 UICSAKeys = false # Default
+# EAL is an experimental feature that exposes API endpoints for reading and writing to blockchains.
+EAL = false # Default
```
Comment (jmank88) [inline core/config/docs/core.toml:17]: Is it an acronym?

File: core/web/router.go [hunk 3]
```diff
@@ -678,3 +679,14 @@ func prometheusHandler(token string, h http.Handler) gin.HandlerFunc {
 		h.ServeHTTP(c.Writer, c.Request)
 	}
 }
+
+func legacyGasStationRoutes(app chainlink.Application, r *gin.RouterGroup) {
+	if app.GetConfig().Feature().EAL() {
+		group := r.Group("/gasstation")
```
Comment (jinhoonbang) [inline core/web/router.go:685]: change to /eal/v1

**Pull request closed without merging**